CHANGELOG.md

```diff
@@ -1,17 +1,24 @@
 ## UNRELEASED
 
 IMPROVEMENTS:
-
+* acl: Remove deprecated `acl_enforce_version_8` option [[GH-7991](https://github.com/hashicorp/consul/issues/7991)]
```

Comment:
I think that should go into `BREAKING CHANGES` - not sure if thats the exact label we used in the past.

Reply:
Fixed